src/platform_impl/windows/event_loop.rs

```diff
@@ -1973,6 +1973,11 @@ unsafe extern "system" fn thread_event_target_callback<T: 'static>(
                 }
             }
 
+            // Default WM_PAINT behaviour. This makes sure modals and popups are shown immediatly when opening them.
```

Comment:
I think this could be replaced then with just calling
```rust
commctrl::DefSubclassProc(window, msg, wparam, lparam)
```

Message boxes are still working correctly then.

Reply:
Did that in the first place but thought ya'll might prefer to call beginpain and endpaint instead.
Changed it to `commctrl::DefSubclassProc(window, msg, wparam, lparam)` in my latest commit

Reply:
One thing that does bother me about this is that it's not clear why `DefSubclassProc` (or `BeginPaint` + `EndPaint` for that matter) isn't being called in the first place. It has apparently been this way since at least #638 (Event Loop 2.0).